awx/api/serializers.py

```diff
@@ -5496,6 +5497,9 @@ def get_field_from_model_or_attrs(fd):
                 raise serializers.ValidationError(
                     "Setting peers manually for control nodes is not allowed. Enable peers_from_control_nodes on the hop and execution nodes instead."
                 )
+                
+        if peers_from_control_nodes and listener_port == None:
```

Comment:
`listener_port is None` might be preferred

also what happens when the input form has

```
"listener_port": ""
```
does it still throw the validation?